packages/create-block/lib/index.js

```diff
@@ -210,6 +199,9 @@ program
 				if ( error instanceof CLIError ) {
 					log.error( error.message );
 					process.exit( 1 );
+				} else if ( error.name === 'ExitPromptError' ) {
+					log.info( 'Please try again later!' );
```

Comment:
```suggestion
					log.info( 'Cancelled.' );
```

Does this message also need to be unified?